config/lighthouse/start-beacon-node.sh

```diff
@@ -25,7 +25,7 @@ exec lighthouse \
         --debug-level $DEBUG_LEVEL \
         --network $NETWORK \
         beacon_node \
-        --eth1-endpoint $VOTING_ETH1_NODE \
+        --eth1-endpoints $VOTING_ETH1_NODE \
```

Comment:
<start-beacon-node.sh> doesn't exist anymore, right?

Reply:
correct, the startup script is now inside the container, look here for details: https://github.com/stereum-dev/ethereum2-docker-build/tree/main/lighthouse/start

I think this is again an outdated commit you see

Reply:
ah yes, thank you